dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -453,8 +453,13 @@
          * This is beacuse we may need to redirect the user to the new URLMap contentlet.
          * We need to wait until the re-index process is done to avoid a 404 error.
          * More info: https://github.com/dotCMS/core/issues/21818
+         *
+         * We also need to wait for the reindex when we edit in order to reload the page when editing using the
+         * #editContentlet() macro
+         * More info: https://github.com/dotCMS/core/issues/30218
+         *
          */
-        const newSaveContentCallBack = isURLMapContent ? (data) => setTimeout(() => saveContentCallback(data), 1800) : saveContentCallback;
+        const newSaveContentCallBack = (data) => setTimeout(() => saveContentCallback(data), 1800);
```

Comment:
Do we agree to delay all edits by `1.8 secs`? This does not happen with regular `contentlets` on the page

Reply:
I wasn't sure about it either, but I don't know how else we can make this. The index is not happening before we reload 😢